sdk/python/readme.py

```diff
@@ -144,6 +145,12 @@ def write_notebook_workflow(
     posix_folder = folder.replace(os.sep, "/")
     posix_notebook = notebook.replace(os.sep, "/")
 
+    # Schedule notebooks at different times to reduce maximum quota usage.
+    name_hash = int(hashlib.sha512(name.encode()).hexdigest(), 16)
```

Comment:
Minor suggestion:
Would it be clearer/better to 
minute_in_interval = hash % (60*hours_between_runs) 
# to ensure uniform distribution , and then derive  
hour = minute_in_interval // 60 
minute = minute_in_interval % 60